test/kwok/kwok.test.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+DIR="$(realpath "${DIR}")"
+
+KUBECTL_VERSION=1.24.2
```

Comment:
do we want to pin the version here?

just wondering

so would this mean that when we add new k8s major version, we also need to update this?

Reply:
I think we can leave it as-is for now, and update each version along with deps bump, or work on a more smart way to autodetect the k8s version.